orchestra/static/orchestra/dashboard/tasktable.html

```diff
@@ -50,8 +50,8 @@
                 <td>{{task.priority}}</td>
                 <td>{{task.project}}</td>
                 <td>{{task.step}}</td>
-                <td>{{task.detail}}</td>
-                <td>{{task.next_todo_description}}</td>
+                <td>{{task.detail|limitTo:50}}{{task.detail.length > 50 ? '...' : ''}}</td>
+                <td>{{task.next_todo_description|limitTo:50}}{{task.next_todo_description.length > 50 ? '...' : ''}}</td>
```

Comment:
Is there pagination? Or does this make it impossible to see > 50 tasks? (i.e., what if Kirstin needs to work on task 51?)

Reply:
this truncates text to 50 characters so the table doesn't wrap